aspnetcore/data/ef-mvc/intro.md

```diff
@@ -220,15 +213,21 @@ In the *Data* folder, create a new class file named *DbInitializer.cs* and repla
 
 The code checks if there are any students in the database, and if not, it assumes the database is new and needs to be seeded with test data.  It loads test data into arrays rather than `List<T>` collections to optimize performance.
 
-In *Startup.cs*, modify the `Configure` method to call this seed method on application startup. First, add the context to the method signature so that ASP.NET dependency injection can provide it to your `DbInitializer` class.
+In *Program.cs*, modify the `Main` method to do the following on application startup:
+
+* Get a database context instance from the dependency injection container.
+* Call the seed method, passing to it the context.
+* Dispose the context when the seed method is done.
 
-[!code-csharp[Main](intro/samples/cu/Startup.cs?name=snippet_ConfigureSignature&highlight=1)]
+[!code-csharp[Main](intro/samples/cu/Program.cs?name=snippet_Seed&highlight=3-20)]
 
-Then call your `DbInitializer.Initialize` method at the end of the `Configure` method.
+Add `using` statements:
 
-[!code-csharp[Main](intro/samples/cu/Startup.cs?name=snippet_RouteAndSeed&highlight=8)]
+[!code-csharp[Main](intro/samples/cu/Program.cs?name=snippet_Usings)]
 
-Now the first time you run the application the database will be created and seeded with test data. Whenever you change your data model, you can delete the database, update your seed method, and start afresh with a new database the same way. In later tutorials you'll see how to modify the database when the data model changes, without deleting and re-creating it.
+Now the first time you run the application, the database will be created and seeded with test data. Whenever you change your data model, you can delete the database, update your seed method, and start afresh with a new database the same way. In later tutorials you'll see how to modify the database when the data model changes, without deleting and re-creating it.
+
+**Note:** For earlier versions of ASP.NET Core, we recommended that you call `DbInitializer.Initialize` in the `Configure` method in *Startup.cs*. In ASP.NET 2.0 we recommend that you use the `Configure` method only to set up the request pipeline. Application startup code, such as `DbInitializer.Initialize`, belongs in the `Main` method. 
```

Comment:
Consider removing the entire note. Do we really need to tell them what we did in the past?